.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: go
+
+go: 1.4
+
+script:
+    - git config --global user.email "you@example.com"
+    - git config --global user.name "Your Name"
```

Comment:
Godep's unit tests run git to synthesize repositories on disk, and git (at least on travis) requires these to be configured.